Bottom: 325a105558bbf3283570812cdd5a2615cce0ffc4
Top:    11d52a6f3a8816b8cd72ad9bff62834e7e979dbf
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-06-05 13:05:08 +0100

Refresh of major-re-write-to-drawing-rout

---

diff --git a/src/draw.h b/src/draw.h
index bb38c20..077c575 100644
--- a/src/draw.h
+++ b/src/draw.h
@@ -76,7 +76,7 @@ void UpdateAll (void);
 
 /* TEMPORARY */
 void DrawPinOrViaLowLevel (PinTypePtr, bool);
-void DrawPlainPin (PinTypePtr, Booleab);
+void DrawPlainPin (PinTypePtr, bool);
 void DrawPlainVia (PinTypePtr, bool);
 void DrawRegularText (LayerTypePtr, TextTypePtr, int);
 void DrawEMark (ElementTypePtr, LocationType, LocationType, bool);
